content/en/users/aai/check-in/service-accounts/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Service Accounts
+linkTitle: Service Accounts
+type: docs
+weight: 50
+description: >
+  Create a Service Account using EGI Check-in
+---
+
```

Comment:
This is currently a deprecated feature. We can document it, but it should explicitly be annotated as deprecated since we do not support it in general availability. @sebastian-luna-valero can you see if there is a Docusaurus feature which allows us to add such a warning or admonition to make it clear to readers that this feature is deprecated and unsupported?

Reply:
Shall we better wait for the resolution of EGIREQ-156 to move forward? If the feature is deprecated and unsupported, and the plan is to stop there, I would not even add this new page.